### PR TITLE
tools: add trouble shooting for binlog

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What is changed, added or deleted? 

add trouble shooting for binlog

### Which version does your change affect? 

only update dev now, will update v2.1, v3.0, and v3.1 after content is ok

### CODE REVIEW

File: dev/reference/tidb-binlog/troubleshoot/error-handling.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+category: reference
+---
+
+# TiDB Binlog 常见错误修复
+
+本文档介绍 TiDB Binlog 中常见的错误以及修复方法。
+
+## Drainer 同步数据到 Kafka 时报错 "kafka server: Message was too large, server rejected it to avoid allocation error"
+
+如果在 TiDB 中执行了大事务，则生成的 binlog 数据会比较大，就可能超过了 Kafka 的消息大小限制。需要调整 Kafka 的配置参数：
```
Comment (yikeke) [inline dev/reference/tidb-binlog/troubleshoot/error-handling.md]:
```suggestion
报错原因：如果在 TiDB 中执行了大事务，则生成的 binlog 数据会比较大，可能超过了 Kafka 的消息大小限制。

解决方法：需要调整 Kafka 的配置参数，如下所示。
```

File: dev/reference/tidb-binlog/troubleshoot/error-handling.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+fetch.message.max.bytes=1073741824
+```
+
+## Pump 报错 "no space left on device"
+
+本地磁盘空间不足，Pump 无法正常写 binlog 数据。需要清理磁盘空间，然后重启 Pump。
+
+## Pump 启动时报错 "fail to notify all living drainer"
+
+Pump 启动时需要通知所有 Online 状态的 Drainer，如果通知失败则会打印该错误日志。可以使用 binlogctl 查看所有 Drainer 的状态是否有异常，保证 Online 状态的 Drainer 都在正常工作。如果某个 Drainer 的状态和实际运行情况不一致，则使用 binlogctl 修改状态，然后再重启 Pump。
```
Comment (yikeke) [inline dev/reference/tidb-binlog/troubleshoot/error-handling.md]:
```suggestion
报错原因：Pump 启动时需要通知所有 Online 状态的 Drainer，如果通知失败则会打印该错误日志。

解决方法：可以使用 binlogctl 工具查看所有 Drainer 的状态是否有异常，保证 Online 状态的 Drainer 都在正常工作。如果某个 Drainer 的状态和实际运行情况不一致，则使用 binlogctl 修改状态，然后再重启 Pump。
```

**Comment by WangXiangUSTC** [comment]
@yikeke thanks, PTAL again

File: dev/reference/tidb-binlog/troubleshoot/error-handling.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+报错原因：本地磁盘空间不足，Pump 无法正常写 binlog 数据。
+
+解决方法：需要清理磁盘空间，然后重启 Pump。
+
+## Pump 启动时报错 "fail to notify all living drainer"
+
+报错原因：Pump 启动时需要通知所有 Online 状态的 Drainer，如果通知失败则会打印该错误日志。
+
+解决方法：可以使用 binlogctl 工具查看所有 Drainer 的状态是否有异常，保证 Online 状态的 Drainer 都在正常工作。如果某个 Drainer 的状态和实际运行情况不一致，则使用 binlogctl 修改状态，然后再重启 Pump。
```
Comment (yikeke) [inline dev/reference/tidb-binlog/troubleshoot/error-handling.md]:
```suggestion
解决方法：可以使用 [binlogctl 工具](/dev/reference/tidb-binlog/maintain.md#binlogctl-工具)查看所有 Drainer 的状态是否有异常，保证 Online 状态的 Drainer 都在正常工作。如果某个 Drainer 的状态和实际运行情况不一致，则使用 binlogctl 修改状态，然后再重启 Pump。
```

**Comment by WangXiangUSTC** [comment]
@yikeke all versions are updated

**Pull request merged**